sdk/src/main/res/values-ja/strings.xml

```diff
@@ -65,6 +65,7 @@
     <string name="payment_method_credit_card_title">クレジットカード</string>
     <string name="payment_method_installment_bay_title">クルンシィ</string>
     <string name="payment_method_installment_bbl_title">バンコク銀行</string>
+    <string name="payment_method_installment_ezypay_title">Maybank Ezypay</string>
```

Comment:
Could you translate this to Japan?

Reply:
I am afraid that we don't have the Japanese translation.

Reply:
I think translation team could help you translate this.

Reply:
thanks to @hpariwat I got a translation.